Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -0,0 +1,536 @@
+    filters = {}
+    for key, value in all_params.items():
+        if key in ['skip', 'limit']:
+            request_data[key] = int(value)
+        if key in ['service', 'instance']:
+            filters[key] = {'eq': int(value)}
+        if key == 'file_type':
+            filters['fileType'] = {'eq': convert_file_type(value)}
+        if key == 'sharing':
+            filters[key] = {'eq': convert_file_sharing(value)}
+        if key == 'extension':
+            filters[key] = {'eq': value}
+        if key == 'quarantined':
+            filters[key] = {'eq': str_to_bool(value)}
+        if key == 'owner':
+            filters['owner.entity'] = {'eq': json.loads(value)}
+    request_data['filters'] = filters
+    return request_data
+
+
+def args_to_json_filter_list_users_accounts(all_params):
+    request_data = {}
+    filters = {}
+    for key, value in all_params.items():
+        if key in ['skip', 'limit']:
+            request_data[key] = int(value)
+        if key in ['app', 'instance']:
+            filters[key] = {'eq': int(value)}
+        if key == 'type':
+            filters[key] = {'eq': value}
+        if key == 'username':
+            filters['entity'] = {'eq': json.loads(value)}
+        if key == 'group_id':
+            filters['userGroups'] = {'eq': value}
+        if key == 'is_admin':
+            filters['isAdmin'] = {'eq': value}
+        if key == 'is_external':
+            filters['isExternal'] = {'eq': convert_is_external(value)}
+        if key == 'status':
+            filters[key] = {'eq': convert_status(value)}
+    request_data['filters'] = filters
+    return request_data
+
+
+def args_to_json_dismiss_and_resolve_alerts(alert_ids, customer_filters, comment):
+    request_data = {}
+    filters = {}
+
+    if alert_ids:
+        id = {'eq': alert_ids.split(',')}
+        filters['id'] = id
+        if comment:
+            request_data['comment'] = comment
+        request_data['filters'] = filters
+    elif customer_filters:
+        request_data.update(json.loads(customer_filters))
+    return request_data
+
+
+def params_to_filter(all_params):
+    filters = {}
+    if 'severity' in all_params.keys():
+        filters['severity'] = {'eq': convert_severity(all_params['severity'])}
+    if 'resolution_status' in all_params.keys():
+        filters['resolutionStatus'] = {'eq': convert_resolution_status(all_params['resolution_status'])}
+    if 'service' in all_params.keys():
+        filters['entity.service'] = {'eq': (int(all_params['service']))}
+    if 'instance' in all_params.keys():
+        filters['entity.instance'] = {'eq': (int(all_params['instance']))}
+    return filters
+
+
+def test_module(client):
+    try:
+        client.alert_list(url_suffix='/alerts/', request_data={"severity": {"eq": 0}})
+    except DemistoException as e:
+        if 'Forbidden' in str(e):
+            return 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
```

Comment:
would be cleaner IMHO
```suggestion
            return(str(e))
```